tmf/org.eclipse.tracecompass.tmf.ui/src/org/eclipse/tracecompass/tmf/ui/viewers/xychart/TmfXYChartViewer.java

```diff
@@ -805,4 +805,13 @@ private void updateTimeFormat() {
             fDataProvider.setTimeFormat(fTimeFormat);
         }
     }
+    /**
+     * @since 9.1
+     * refresh fMouseSelectionProvider
+     */
+
+    public void refreshMouseSelectionProvider() {
```

Comment:
I do not think this change is justified, the selectionRangeUpdated should update the mouse selection provider.